modules/afsocket/afsocket-dest.c

```diff
@@ -353,9 +353,11 @@ afsocket_dd_setup_addresses_method(AFSocketDestDriver *self)
 }
 
 static void
-afsocket_dd_restore_connection(AFSocketDestDriver *self)
+afsocket_dd_restore_writer(AFSocketDestDriver *self)
 {
-  GlobalConfig *cfg = log_pipe_get_config(&self->super.super.super);
+  GlobalConfig *cfg;
```

Comment:
oh... f***k made a mistakde during conflict resolution, please do not merge until I fixed it :-)
(cfg is not initialized!!!)